src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -20,21 +20,88 @@ export default class StickyToolbarView extends ToolbarView {
 	/**
 	 * @inheritDoc
```

Comment:
As to

> Besides, can't we move the limiter to the model?

please see https://github.com/ckeditor/ckeditor5-editor-classic/pull/30/files/4a846ef87e8876049dfa233a5b0f7b94f5d0eadf#r78907966.